.github/.gitmojis.json

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
We should only have gitmojis we are using instead of all the others.
This will ensure that not permitted gitmojis in PR name will not passed.

Reply:
Done.